.github/workflows/build.yml

```
@@ -25,7 +25,7 @@ jobs:
 
     - uses: actions/setup-java@v2
       with:
-        distribution: 'adopt'
+        distribution: 'zulu'
```

```suggestion
        distribution: 'temurin'
```
See comment. We prefer to use Eclipse Temurin

---

I've updated both workflows to use Temurin as the distribution for setup-java GH actions. 👍🏼